src/DynamicExpresso.Core/Parsing/Parser.cs

```diff
@@ -3340,5 +3336,73 @@ private static T[] RemoveLast<T>(T[] array)
 				return result;
 			}
 		}
+
+		/// <summary>
+		/// Binds to a member access of an instance as late as possible.  This allows the use of anonymous types on dynamic values.
+		/// </summary>
+		private class LateGetMemberCallSiteBinder : CallSiteBinder
+		{
+			private readonly string m_propertyOrFieldName;
+
+			public LateGetMemberCallSiteBinder(string propertyOrFieldName)
+			{
+				m_propertyOrFieldName = propertyOrFieldName;
+			}
+
+			public override Expression Bind(object[] args, ReadOnlyCollection<ParameterExpression> parameters, LabelTarget returnLabel)
+			{
+				var binder =
+					Microsoft.CSharp.RuntimeBinder.Binder.GetMember(
```

Comment:
[NIT] `var binder = Microsoft.CSharp.RuntimeBinder.Binder.GetMember(` on the same line

Reply:
Resolved in latest commit